ceph_deploy/conf/cephdeploy.py

```diff
@@ -37,7 +37,7 @@
 # enabled=1
 # gpgcheck=1
 # type=rpm-md
-# gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
+# gpgkey=https://git.ceph.com/?p=ceph.git;a=blob_plain;f=keys/release.asc
```

Comment:
Ack - yes we can.  For some reason I had this in my head as "docs", like the .rst files. Silly of me...  :) Nice catch!